tasks/gaia.js

```diff
@@ -275,13 +276,25 @@ function makeExecWithInputs(command, inputs = [], json = true) {
     })
 
     child.once(`exit`, code => {
+      if (process.env.VERBOSE) {
+        console.log(`EXIT:`, code)
+      }
       if (resolved) return
       resolved = true
       code === 0 ? resolve() : reject(`Process exited with code ${code}`)
     })
   })
 }
 
+/*
+
+    mnemonic: `release endorse scale across absurd trouble climb unaware actor elite fantasy chair license word rare length business kiss smoke tackle report february bid ginger`,
+    address: `cosmos1ek9cd8ewgxg9w5xllq9um0uf4aaxaruvcw4v9e`,
+    */
+function sendMoneyToFixedAccounts(mainAccountSignInfo, chainId) {
+  sendTokens(mainAccountSignInfo, `${100 * 10e6}stake`, `cosmos1ek9cd8ewgxg9w5xllq9um0uf4aaxaruvcw4v9e`, chainId)
```

Comment:
I would just move `cosmos1ek9cd8ewgxg9w5xllq9um0uf4aaxaruvcw4v9e` to a variable, I've seen it's copy pasted in a couple of places